### DISCUSSION TITLE
How to properly add the Electron based app?

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5021] [Help and Questions] [answered]
Hello!

I was trying to add GDevelop ( https://gdevelop.io/ ) to my bucket list, but i cannot properly figure out how to extract/install the downloaded EXE and make the correct manifest for it.. (shortcuts, bins, ...) for it to work as expected in the end.

Currently i have 'gdevelop.json' in my bucket folder that has:

```
{
    "homepage": "https://gdevelop.io/",
    "version": "5.0.138",
    "description": "GDevelop is an open-source, cross-platform game engine designed to be used by everyone.",
    "license": "MIT",
    "hash": "b16c428028d9538fb0e9759603b958e066fb3674435503cbf8da63d44f84dcfe",
    "url": "https://github.com/4ian/GDevelop/releases/download/v5.0.138/GDevelop-5-Setup-5.0.138.exe",
   
    "checkver": {
        "github": "https://github.com/4ian/GDevelop/releases/download/v5.0.138/GDevelop-5-Setup-5.0.138.exe"
    },
    "autoupdate": {
        "url": "https://github.com/4ian/GDevelop/releases/download/$version/GDevelop-5-Setup.exe"
    }
}
```

It seems to work properly, with exceptions that the file it downloads is just the installer, which i do not know how to extract (it it Electron based packaged app, while opened up with 7Zip it has '$PLUGINSDIR' and app64.7z file in it, that's how i know that it is Electron based.) and the second exception being the 'bins' and 'shortcuts' that upon reading any reading manual / material that is available, still cannot figure it out..

**Comment by niheaven** [comment, marked as answer]
Search extras bucket for `$PLUGINSDIR`, you'll find many Electron-based apps, and use them as templates.

**Reply by rashil2000** [reply]
Refer to https://github.com/ScoopInstaller/Extras/search?q=App-64&type=

**Comment by anderlli0053** [comment]
Thanks both of you 

@niheaven  @rashil2000
